README.md

```diff
@@ -119,6 +118,46 @@ end
 
 ```
 
+### Go apps
+
+Add the following handler to your app:
+
+```go
+http.HandleFunc("/.well-known/acme-challenge/", func(w http.ResponseWriter, r *http.Request) {
+  pt := strings.TrimPrefix(r.URL.Path, "/.well-known/acme-challenge/")
+  rk := ""
```

Comment:
This is a nitpick, but the Go convention when setting vars which aren't used right away is to use `var rk string`.